### ISSUE TITLE
Complete draft Vocabulary Maintenance Specification

### ISSUE THREAD

**Issue by baskaufs** [tdwg/vocab#2]
This specification will be focused on the process by which standards that are vocabularies are modified and amended.  The Term Change Policy of the Darwin Core (DwC) Namespace Policy (http://rs.tdwg.org/dwc/terms/namespace/index.htm#classesofchanges) will be a starting point.  The DwC Term Change Policy would ultimately be replaced by this specification, which would apply not only to Darwin Core but to all TDWG vocabularies that are Technical Specification standards.

**Comment by baskaufs** [comment]
Blocked on Issue #6

**Comment by baskaufs** [comment]
Blocked on Issue #7

**Comment by baskaufs** [comment]
Blocked on Issue #9

**Comment by baskaufs** [comment]
Blocked on Issue #11

**Comment by baskaufs** [comment]
Blocked on Issue #14

**Comment by baskaufs** [comment]
Blocked by Issue #17

**Comment by baskaufs** [comment]
Blocked by Issue #18

**Comment by baskaufs** [comment]
Blocked by Issue #19

**Comment by baskaufs** [comment]
Blocked by Issue #21

**Comment by baskaufs** [comment]
Refer to [Publishing a Vocabulary as a SISSvoc service](https://www.seegrid.csiro.au/wiki/Siss/SISSvoc30VocabularyDeployment) for a workflow on versioning and updating vocabularies, as well as making them available via SPARQL, URI dereferencing, and publishing the vocabulary as a single document.

**Comment by baskaufs** [comment]
No longer blocked by Issues #6, #7, #11, #14, #17, #19, and #21, which are now closed.  I've left Issue #9 open because I'm not sure that it is really resolved.  However, I think that whatever parts of that issue have not yet been resolved are out of scope for our task.  So I've removed the block tag from this issue.

**Comment by baskaufs** [comment]
I have completed the final proofreading edits based on the comments I've gotten since the Task Group hangout.  Unless there is something really bad that needs to be fixed, I'm going to call the draft ready to submit with a request for appointment of the review editor.
